packages/xo-web/src/common/store/reducer.js

```diff
@@ -92,6 +93,23 @@ export default {
       homeVmIdsSelection,
   }),
 
+  /**
+   * This state is used temporarily to keep loading state of template installation in hub
+   * @param {object} hubInstallingResources A map of { <templateId>: <loadingState }
+   * @param {boolean} hubInstallingResources[<templateId>] true: loading / false: no operation
+   */
```

Comment:
Simplify:

```js
// whether a resource is currently being installed: `hubInstallingResources[<template id>]`
```

I think that's enough to understand how to use it :slightly_smiling_face: